xAPI.md

```diff
@@ -507,6 +507,47 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
+This section explains and shows graphically how different pieces of xAPI can fit together.  
+
+##### Tracking Experiences Through Systems
+
+![Data Flow in xAPI](/xAPIDataFlow.jpg)
+
+Figure 1 shows the tracking of experiences through the data flow of xAPI. A user either accesses on their own, or 
+is provided access to web-enabled content.  The user, known by the LRS, is tracked in the experiences that he or 
+she has while taking that content. A Data Analysis Tool, acting as a Service with permission, can grab the data  
+from the LRS and use it to display meaningful reports or visualizations to the user. In the case of more 
+structured   learning, a user may be dependent on an Activity Provider to deliver and/or authorize use of content 
+to them.  Once this triangle of authorization and access is complete, a registration exists for that user in that 
+content tracked by that LRS and experience tracking may proceed.
```

Comment:
I didn't see this bit before suggesting the text change above. If you go with the text change i suggested maybe this bit gets removed?